### DISCUSSION TITLE
Rename label of Dark Mode

### DISCUSSION THREAD

**Discussion by mrvectopus** [shuding/nextra#2604] [Q&A] [answered]
👋 Hello!

This it's possible to change a default labels of DarkMode? I mean rename this labels: `"System"`  `"Light"` ` "Dark"`

The `i18n` do not translate this labels :(

**Comment by jellli** [comment]
Hi, [this](https://github.com/shuding/nextra/issues/2584) issue may be helpful

**Reply by mrvectopus** [reply]
Thank you! But this not help or maybe I'm use incorrect setting...

<img width="1260" alt="Screenshot 2023-12-29 в 7 11 32" src="https://github.com/shuding/nextra/assets/153316370/8858f6f6-ccd7-47bd-bb52-87da998dc2f5">

<img width="1273" alt="Screenshot 2023-12-29 в 7 11 36" src="https://github.com/shuding/nextra/assets/153316370/6f132465-8e37-4175-beba-47b8c39be19d">

**Reply by jellli** [reply, marked as answer]
Have you tried to update your docs theme?

**Reply by mrvectopus** [reply]
Hm... I'm use this template https://github.com/shuding/nextra-docs-template and he last update - 10 months ago 😄 

I'm go to try install Nextra without this template 👨‍💻 

UPD. After upgrade this method is work! 👍
